### ISSUE TITLE
Mismatched SemanticVersion

### ISSUE THREAD

**Issue by mosesprime** [natecraddock/ziglua#128]
There seems to be a mismatch in the lua 5.4 versioning between the build and the zon.

https://github.com/natecraddock/ziglua/blob/90dab7e72173709353dcaaa6d911bed7655c030d/build/lua.zig#L16-L37

vs

https://github.com/natecraddock/ziglua/blob/90dab7e72173709353dcaaa6d911bed7655c030d/build.zig.zon#L26-L29

I am still new to zig, so please forgive my naivete, but is defining the semantic version in build/lua.zig strictly necessary if the version is already controlled by build.zig.zon?

**Comment by robbielyman** [comment]
good catch, thanks! i think the version is not core to the functioning of any part of the module, but it might be useful for somebody using ziglua as a dependency.
